.github/workflows/continous-integration-docker.yml

```
@@ -509,7 +508,7 @@ jobs:
     name: lightwalletd tip send
     needs: lightwalletd-full-sync
     uses: ./.github/workflows/deploy-gcp-tests.yml
-    if: ${{ !cancelled() && !failure() && github.event.inputs.regenerate-disks != 'true' && github.event.inputs.run-full-sync != 'true' && github.event.inputs.run-lwd-sync != 'true' }}
+    if: ${{ !cancelled() && !failure() && github.event_name == 'push' && github.ref_name == 'main' && github.event.inputs.regenerate-disks != 'true' && github.event.inputs.run-full-sync != 'true' && github.event.inputs.run-lwd-sync != 'true' }}
```

I'm not sure if some of these conditions are needed.

The manual workflow inputs will always have their default values during a `push` event, because they are only used for `workflow_dispatch` events.

```suggestion
    if: ${{ !cancelled() && !failure() && github.event_name == 'push' && github.ref_name == 'main' }}
```

---

Good point, thank you.